curriculum/challenges/english/14-responsive-web-design-22/learn-css-colors-by-building-a-set-of-colored-markers/61764c602bee6974e7790f35.md

```diff
@@ -13,6 +13,14 @@ These days, there are two main color models: the <dfn>additive</dfn> RGB (red, g
 
 First, add the `class` `one` to the first marker `div` element.
 
+Here is the syntax to add multiple classes:
```

Comment:
Although I like your wording, it's extra work for the translation team. 

 Unfortunately, I think, it's better to not add it.

Reply:
Doesn't camperbot process the translations?

Reply:
Oh, the translation is done by contributors in crowdin, the bot is responsible for getting it from the platform. If I am not misunderstanding.

https://contribute.freecodecamp.org/#/how-to-translate-files